### PR TITLE
New structure

### PR DESCRIPTION
**Pull request by keroloussamy**
Add the new structure and update old controllers

FC

### CODE REVIEW

File: app/controllers/api/room_types_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Api
+  class RoomTypesController < ApplicationController
+    def index
+      @room_types = RoomType.all
+      render json: @room_types
+    end
+  end
+end
```
Comment (Lucas-Masaba) [inline app/controllers/api/room_types_controller.rb:8]: - [ ] Please add an endpoint for room_types in `routes.rb`.  👍

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @keroloussamy  , 

Good job so far! 👍 

There is one more change you need to make for this to et approved. Check the comments below

Cheers and Happy coding!👏👏👏

File: app/controllers/api/room_types_controller.rb (hunk 1 above)
Comment (keroloussamy) [reply app/controllers/api/room_types_controller.rb:8]: I have added it on the second PR #30

**Review by Lucas-Masaba — APPROVED** [review]
Hello @keroloussamy  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

Cheers and Happy coding!👏👏👏 💻

**Pull request merged**